source/Body.h

```diff
@@ -18,21 +18,27 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 
 #include "Angle.h"
 #include "Point.h"
+#include "SpriteParameters.h"
 
 #include <cstdint>
+#include <map>
 #include <string>
 
 class DataNode;
 class DataWriter;
 class Government;
 class Mask;
+class Outfit;
```

Comment:
Not a great fan of involving the outfit class in the body class, but I guess if you need it, I am somewhat ok with it...

Reply:
This has also bothered me, and I am thinking of workarounds. I remember seeing something about conditionals in the datafiles, maybe that could be a better alternative.

The reason I need outfits currently is due to trigger sprites (e.g. if "Sunbeam" is installed then play this sprite, otherwise, play another sprite). But with conditionals in the datafiles, I could just check if <condition> is true, then play this sprite, otherwise, play another sprite. I will look into it.

Reply:
@RisingLeaf Made the switch, the Body class no longer depends on the outfit class. Now the Body class has its own ConditionsStore which can be populated by the objects implementing body. These conditions are now used to trigger different sprites. I have outlined how that works in the description.